lib/web_ui/dev/goldens.dart

```diff
@@ -33,10 +42,60 @@ class ImageDiff {
   int _pixelCount = 0;
   int _wrongPixels = 0;
 
+  static final double _maxTheoreticalColorDistance = Color.distance(
+    <int>[255, 255, 255],
+    <int>[0, 0, 0],
+    false,
+  );
+
+  // If the normalized color difference of a pixel is greater than this number,
+  // we consider it a wrong pixel.
+  static const double _kColorDistanceThreshold = 0.1;
+
   final int _colorOk = Color.fromRgb(255, 255, 255);
   final int _colorBadPixel = Color.fromRgb(255, 0, 0);
   final int _colorExpectedPixel = Color.fromRgb(0, 255, 0);
 
+  static int _reflectedPixel(Image image, int x, int y) {
+    x = x.abs();
+    if (x == image.width) {
+      x = image.width - 2;
```

Comment:
Maybe a comment on what this does and why it's necessary?

Reply:
Done. Added a dart doc explaining what "reflecting" means.